sfepy/discrete/fem/meshio.py

```diff
@@ -403,6 +403,11 @@ def write(self, filename, mesh, out=None, ts=None, **kwargs):
         if self.file_format in ['vtk', 'vtu']:
             ngkey = 'node_groups'
             cgkey = 'mat_id'
+            if coors.shape[1] != 3:
+                nnd, ndim = coors.shape
+                ndim = 3 - ndim
+                coors = nm.hstack([coors,
```

Comment:
meshio uses `numpy.pad()` - might be faster?

Reply:
I hope that this way is fast enough. From my point of view, this solution is more readable than   using `numpy.pad()`.

Reply:
OK